TESTS.rst

```diff
@@ -1,19 +1,54 @@
 Software tests
```

Comment:
Is this file linked anywhere else in docs?

Reply:
No, this is purely a top level readme-esque file. This used to be in the `README.rst` file, but we spun it out into it's own file so it wouldn't render on github and take too much space.

Reply:
But, the same information is also duplicated inside the docs (in the installation page).